python/pfs/drp/stella/fitPfsFluxReference.py

```diff
@@ -1837,6 +1856,75 @@ def marginalizePdf(
     return y, marginalizedPdf
 
 
+def isMidResolution(pfsMerged: PfsMerged) -> bool:
+    """Return True if a part of spectra in ``pfsMerged`` are from
+    mid-resolution arm.
+
+    This function guesses used arms from the observed spectra instead of
+    referring to ``pfsConfig.arms`` because ``pfsConfig.arms`` is unreliable.
+
+    Parameters
+    ----------
+    pfsMerged : `PfsMerged`
+        Merged spectra from exposure.
+
+    Returns
+    -------
+    midres : `bool`
+        True if a part of spectra in ``pfsMerged`` are from mid-resolution arm.
+    """
+    return any("m" in arms for arms in getExistentArms(pfsMerged).values())
```

Comment:
Why not use `"m" in pfsMerged.identity.arm`?

Reply:
Thanks for the information. I will use `pfsMerged.identity.arm` here, and move `getExistentArms()` to `tickets/PIPE2D-1427`